pkg/api/plugins_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/grafana/grafana/pkg/plugins"
 	"github.com/grafana/grafana/pkg/plugins/auth"
 	"github.com/grafana/grafana/pkg/plugins/config"
-	"github.com/grafana/grafana/pkg/plugins/manager/fakes"
+	pluginfakes "github.com/grafana/grafana/pkg/plugins/manager/fakes"
```

Comment:
You reminded me of something I have wanted to do for ages: https://github.com/grafana/grafana/pull/112503. I'm not sure how many fakes there are within `pkg/services/pluginsintegration`, but would be a good idea to bundle them all together in a similar package there too if we can 🙌